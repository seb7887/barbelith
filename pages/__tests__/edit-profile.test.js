import 'react-testing-library/cleanup-after-each';

import { render } from 'react-testing-library';
import EditProfilePage from '../edit-profile';

const auth = {
  user: {
    _id: '7'
  }
};

const mockedProps = {
  auth
};

describe('<EditProfilePage />', () => {
  it('renders and matches snapshot', () => {
    const { asFragment } = render(
      <EditProfilePage {...mockedProps} />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});