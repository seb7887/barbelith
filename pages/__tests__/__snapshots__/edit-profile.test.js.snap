// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EditProfilePage /> renders and matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="EditProfile-root-1"
  >
    <div
      class="MuiPaper-root-11 MuiPaper-elevation2-15 MuiPaper-rounded-12 EditProfile-paper-4"
    >
      <div
        class="MuiAvatar-root-38 MuiAvatar-colorDefault-39 EditProfile-avatar-5"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root-41"
          focusable="false"
          role="presentation"
          viewBox="0 0 24 24"
        >
          <path
            d="M0 0h24v24H0V0z"
            fill="none"
          />
          <g>
            <path
              d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM21.41 6.34l-3.75-3.75-2.53 2.54 3.75 3.75 2.53-2.54z"
            />
          </g>
        </svg>
      </div>
      <h1
        class="MuiTypography-root-50 MuiTypography-h5-66"
      >
        Edit Profile
      </h1>
      <form
        class="EditProfile-form-6"
      >
        <div
          class="MuiAvatar-root-38 MuiAvatar-colorDefault-39 EditProfile-bigAvatar-2"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root-41"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M0 0h24v24H0V0z"
              fill="none"
            />
            <g>
              <path
                d="M17.5 8c.46 0 .91-.05 1.34-.12C17.44 5.56 14.9 4 12 4c-.46 0-.91.05-1.34.12C12.06 6.44 14.6 8 17.5 8zM8.08 5.03C6.37 6 5.05 7.58 4.42 9.47c1.71-.97 3.03-2.55 3.66-4.44z"
                opacity=".3"
              />
              <path
                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 2c2.9 0 5.44 1.56 6.84 3.88-.43.07-.88.12-1.34.12-2.9 0-5.44-1.56-6.84-3.88.43-.07.88-.12 1.34-.12zM8.08 5.03C7.45 6.92 6.13 8.5 4.42 9.47 5.05 7.58 6.37 6 8.08 5.03zM12 20c-4.41 0-8-3.59-8-8 0-.05.01-.1.01-.15 2.6-.98 4.68-2.99 5.74-5.55 1.83 2.26 4.62 3.7 7.75 3.7.75 0 1.47-.09 2.17-.24.21.71.33 1.46.33 2.24 0 4.41-3.59 8-8 8z"
              />
              <circle
                cx="9"
                cy="13"
                r="1.25"
              />
              <circle
                cx="15"
                cy="13"
                r="1.25"
              />
            </g>
          </svg>
        </div>
        <input
          accept="image/*"
          class="EditProfile-input-10"
          data-testid="avatar"
          id="avatar"
          name="avatar"
          type="file"
        />
        <label
          class="EditProfile-uploadButton-3"
          for="avatar"
        >
          <span
            class="MuiButtonBase-root-112 MuiButton-root-86 MuiButton-contained-97 MuiButton-containedSecondary-99 MuiButton-raised-100 MuiButton-raisedSecondary-102"
            role="button"
            tabindex="0"
          >
            <span
              class="MuiButton-label-87"
            >
              Upload Image 
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root-41"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M0 0h24v24H0z"
                  fill="none"
                />
                <path
                  d="M19.35 10.04C18.67 6.59 15.64 4 12 4 9.11 4 6.6 5.64 5.35 8.04 2.34 8.36 0 10.91 0 14c0 3.31 2.69 6 6 6h13c2.76 0 5-2.24 5-5 0-2.64-2.05-4.78-4.65-4.96zM14 13v4h-4v-4H7l5-5 5 5h-3z"
                />
              </svg>
            </span>
            <span
              class="MuiTouchRipple-root-183"
            />
          </span>
        </label>
        <span />
        <div
          class="MuiFormControl-root-115 MuiFormControl-marginNormal-116 MuiFormControl-fullWidth-118"
        >
          <label
            class="MuiFormLabel-root-130 MuiFormLabel-required-135 MuiInputLabel-required-123 MuiInputLabel-root-119 MuiInputLabel-formControl-124 MuiInputLabel-animated-127"
            data-shrink="false"
            for="name"
          >
            Name
            <span
              class="MuiFormLabel-asterisk-136"
            >
               *
            </span>
          </label>
          <div
            class="MuiInputBase-root-150 MuiInput-root-137 MuiInput-underline-141 MuiInputBase-formControl-151 MuiInput-formControl-138"
          >
            <input
              aria-invalid="false"
              class="MuiInputBase-input-160 MuiInput-input-145"
              id="name"
              name="name"
              required=""
              type="text"
              value=""
            />
          </div>
        </div>
        <div
          class="MuiFormControl-root-115 MuiFormControl-marginNormal-116 MuiFormControl-fullWidth-118"
        >
          <label
            class="MuiFormLabel-root-130 MuiFormLabel-required-135 MuiInputLabel-required-123 MuiInputLabel-root-119 MuiInputLabel-formControl-124 MuiInputLabel-animated-127"
            data-shrink="false"
            for="about"
          >
            About
            <span
              class="MuiFormLabel-asterisk-136"
            >
               *
            </span>
          </label>
          <div
            class="MuiInputBase-root-150 MuiInput-root-137 MuiInput-underline-141 MuiInputBase-formControl-151 MuiInput-formControl-138"
          >
            <input
              aria-invalid="false"
              class="MuiInputBase-input-160 MuiInput-input-145"
              id="about"
              name="about"
              required=""
              type="text"
              value=""
            />
          </div>
        </div>
        <div
          class="MuiFormControl-root-115 MuiFormControl-marginNormal-116 MuiFormControl-fullWidth-118"
        >
          <label
            class="MuiFormLabel-root-130 MuiFormLabel-required-135 MuiInputLabel-required-123 MuiInputLabel-root-119 MuiInputLabel-formControl-124 MuiInputLabel-animated-127"
            data-shrink="false"
            for="email"
          >
            Email
            <span
              class="MuiFormLabel-asterisk-136"
            >
               *
            </span>
          </label>
          <div
            class="MuiInputBase-root-150 MuiInput-root-137 MuiInput-underline-141 MuiInputBase-formControl-151 MuiInput-formControl-138"
          >
            <input
              aria-invalid="false"
              class="MuiInputBase-input-160 MuiInput-input-145 MuiInputBase-inputType-163 MuiInput-inputType-148"
              id="email"
              name="email"
              required=""
              type="email"
              value=""
            />
          </div>
        </div>
        <button
          class="MuiButtonBase-root-112 MuiButtonBase-disabled-113 MuiButton-root-86 MuiButton-contained-97 MuiButton-containedSecondary-99 MuiButton-raised-100 MuiButton-raisedSecondary-102 MuiButton-disabled-106 MuiButton-fullWidth-111 EditProfile-submit-7"
          data-testid="submit"
          disabled=""
          tabindex="-1"
          type="submit"
        >
          <span
            class="MuiButton-label-87"
          >
            Save
          </span>
        </button>
      </form>
    </div>
  </div>
</DocumentFragment>
`;
